script/DeployMocks.s.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: BUSL 1.1
+// Valorem Labs Inc. (c) 2023.
+pragma solidity 0.8.16;
+
+import "forge-std/Script.sol";
+
+// import "../src/TokenURIGenerator.sol";
+import "test/mocks/MockUSDC.sol";
+import "test/mocks/MockWETH.sol";
+import "test/mocks/MockGMX.sol";
+
+
```

Comment:
Nit: run `npm run lint` to clean up formatting. This should remove extra lines and such, and get the CI build passing.